apps/commudle-admin/src/app/app.component.ts

```diff
@@ -37,14 +37,18 @@ export class AppComponent implements OnInit, AfterViewChecked, OnDestroy {
     private profileStatusBarService: ProfileStatusBarService,
     private isBrowserService: IsBrowserService,
     private seoService: SeoService,
-    private router: Router
+    private router: Router,
   ) {
     this.apiRoutes.setBaseUrl(environment.base_url);
     this.actionCableConnectionSocket.setBaseUrl(environment.anycable_url);
     this.isBrowser = this.isBrowserService.isBrowser();
   }
 
   ngOnInit(): void {
+    const host = window.location.hostname;
+    if (['test'].includes(host)) {
```

Comment:
this should be very specific, we need to mention that the host should be test.commudle.com
a project name could also be test, or a lab or a community....

Reply:
In fact this code is not needed.